src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -309,7 +310,7 @@ public static List<RelatedParty> getIndividualRelatedParties(Entity entity, Stri
         switch (roleCode) {
             case APPOINTEE:
```

Comment:
You'll want to add relatedParties.add(getRelatedParty(partyId, roleCode.getHmcReference())); here for APPOINTEE

Reply:
You are right otherwise always related party would be SOLICITOR for also APPOINTEE. 
Corrected it.